tests/functional/test_override_ssl_common_name.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from awscli.testutils import mock, create_clidriver
+import pytest
+
+from awscli.customizations.overridesslcommonname import ssl_common_names
+
+regions_to_exclude = [
```

Comment:
What was the reason for excluding some of the regions?

Another approach may be to map services to the operation and grab everything.

````python
def common_name_test_cases():

    service_ops = {
        "sqs": "list-queues",
        "emr": "list-clusters" ,
        "shield": "list-protections" ,
        "rds": "describe-db-clusters",
    }
    for service in ssl_common_names.keys():
        for region in ssl_common_names[service]:
            yield (service, service_ops[service], region)